console/src/components/Services/Data/Add/TableDefault.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import CustomInputAutoSuggest from '../../../Common/CustomInputAutoSuggest/CustomInputAutoSuggest';
+import {
+  getPlaceholder,
+  getDefaultValue,
+  getDefaultFunctionsOptions,
+} from '../Common/utils';
+
+const TableDefault = ({
+  column,
+  colDefaultFunctions,
+  onChange,
+  testId,
+  colIndex: i,
+}) => {
+  // const styles = require('../../../Common/TableCommon/Table.scss');
+  const defaultInputTheme = require('../../../Common/CustomInputAutoSuggest/CustomInputAddTableTheme.scss');
```

Comment:
Here is the problem: 

In order to style the auto suggest input the [library owner](https://github.com/moroshko/react-autosuggest) provides multiple ways, one of the way is to use css modules since we have used it extensively everywhere. 

It accepts a `theme` prop which basically is a compiled json object of the .scss file. Now in order to customize the css based on the requirement there is a file @ src/components/Common/CustomInputAutoSuggest/CustomInputTheme.scss which is sort of like  a basic layout but it can be tweaked based on the requirement. Now inorder to do that you import it in your .scss file and modify the required properties. Should we maintain a seperate copy at each place the component is used or just move the file somewhere where it designates a common .scss file like **components/Common**?

@rikinsk Let me know what you think. I have fixed added of the suggestions